tools/clean-webpack.ts

```diff
@@ -0,0 +1,4 @@
+import * as fs from 'node:fs';
+import * as path from 'node:path';
+
+fs.rmSync(path.join(__dirname, '.webpack'), { force: true, recursive: true });
```

Comment:
`__dirname`-free alternative:
```suggestion
fs.rmSync(path.resolve('.webpack'), { force: true, recursive: true });
```

Reply:
Yea, the only reason I went with the `__dirname` version was I wanted a little bit of extra protection so it won't go deleting `${CWD}/.webpack` if a user inadvertently invoked it from a different CWD. Probably not an actual concern, but I don't love blindly deleting a relative path.